### ISSUE TITLE
Error fetching from carbon: TypeError: Error processing argument at index 0, conversion failure from undefined

### ISSUE THREAD

**Issue by thcolin** [npezza93/carbon#1]
Hi !

I tried your plugin but it doesn't seems to work, here's returned Atom error : 

**Atom**: 1.29.0 x64
**Electron**: 2.0.5
**OS**: Mac OS X 10.13.6
**Thrown From**: Atom Core

### Steps to reproduce

1. Select some code in Atom
2. `Copy code screenshot to clipboard` from context menu

### Stack Trace

Error fetching from carbon: TypeError: Error processing argument at index 0, conversion failure from undefined

```
At undefined

undefined
```

### Commands

```
     -2:02.8.0 debugger:show-attach-dialog (input.hidden-input)
     -2:01.5.0 editor:consolidate-selections (input.hidden-input)
     -2:01.5.0 core:cancel (input.hidden-input)
     -1:59 editor:select-to-first-character-of-line (input.hidden-input)
     -1:55.7.0 carbon.now.sh:toggle (div.line)
     -1:35.8.0 debugger:show-attach-dialog (input.hidden-input)
     -1:34.5.0 editor:consolidate-selections (input.hidden-input)
     -1:34.5.0 core:cancel (input.hidden-input)
     -1:33.7.0 debugger:show-attach-dialog (input.hidden-input)
     -1:31.1.0 editor:consolidate-selections (input.hidden-input)
     -1:31.1.0 core:cancel (input.hidden-input)
     -1:27.9.0 carbon.now.sh:toggle (div.line)
  2x -1:23.2.0 core:close (input.hidden-input)
     -0:44.3.0 settings-view:open (input.hidden-input)
     -0:40.9.0 core:confirm (input.hidden-input)
     -0:02.9.0 carbon:copy (span.syntax--string.syntax--quoted.syntax--double.syntax--twig)
```

### Non-Core Packages

```
atom-ide-ui 0.13.0 
atom-quokka 1.0.79 
better-git-blame 0.4.6 
busy-signal 1.4.3 
carbon 0.3.0 
dark-one-dark-syntax 2.3.0 
dracula-syntax 2.0.7 
dracula-ui 0.6.0 
editorconfig 2.2.2 
highlight-selected 0.14.0 
ide-typescript 0.7.6 
intentions 1.1.5 
language-babel 2.85.0 
language-mdx 0.1.0 
language-twig 1.6.3 
linter 2.2.0 
linter-ui-default 1.7.1 
pigments 0.40.2 
prettier-atom 0.54.0 
todo-show 2.3.0 
```

**Comment by npezza93** [comment]
Thanks for the report! I'll check it out

**Comment by abdul-elah-js** [comment]
any solution yet ?

**Comment by Beyarz** [comment]
No

**Comment by npezza93** [comment]
Fixed in v0.3.1
